mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -18,11 +18,16 @@ pipeline {
                 sh '''
                     # install pkg dependencies
                     sudo apt update
-                    sudo apt install -y ninja-build wget git python cmake
+                    sudo apt install -y ninja-build wget git python3.7 cmake python3-pip
+		    if [ -f /usr/bin/python ]; then
+                      	/bin/rm /usr/bin/python
+                    fi 
+		    ln -s /usr/bin/python3.7 /usr/bin/python
+	 	    python3.7 -m pip install pathspec unidiff
                 '''
             }
         }
-        stage('Test') {
+        stage('Static Test') {
```

Comment:
Thanks for making static test as an independent stage. Though github still shows everything as a single item. Maybe a good time to ask the MIOpen channel about how they did it?

Reply:
It seems that showing build status for each stage is done via a Jenkins plugin github-autostatus. There is no need to change the jenkinsfile. However I couldn't find this plugin in our Jenkins system.